Sources/Markdown/Markdown.docc/Markdown.md

```diff
@@ -36,3 +36,5 @@ The markup tree provided by this package is comprised of immutable/persistent, t
 
 - <doc:VisitMarkup> 
 - <doc:FormatterAndOptions>
+
+<!-- Copyright (c) 2021 Apple Inc and the Swift Project authors. All Rights Reserved. -->
```

Comment:
Should this one's copyright years be updated, even if the update that happened this year was the addition of the copyright line itself? 🤔 Probably fine either way.